workers/unity/Assets/Gdk/Core/GameObjectRepresentation/GameObjectDispatcherCallbacksSystem.cs

```diff
@@ -0,0 +1,70 @@
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core
+{
+    /// <summary>
+    ///     Gathers incoming dispatcher ops and invokes callbacks on relevant GameObjects.
+    /// </summary>
+    [UpdateInGroup(typeof(SpatialOSUpdateGroup))]
+    public class GameObjectDispatcherCallbacksSystem : ComponentSystem
+    {
+        private MutableView view;
+
+        protected override void OnCreateManager(int capacity)
+        {
+            base.OnCreateManager(capacity);
+
+            var worker = WorkerRegistry.GetWorkerForWorld(World);
+            view = worker.View;
+
+            GenerateComponentGroups();
+        }
+
+        private void GenerateComponentGroups()
+        {
+            foreach (var gameObjectTranslation in view.GameObjectTranslations)
```

Comment:
If this is the only place these are used - find them in here rather the Mutable View

Reply:
Moved the GameObjectTranslationsDictionary into the system.